podcasts/ShelfActionsViewController.swift

```diff
@@ -148,6 +157,14 @@ class ShelfActionsViewController: UIViewController {
         actionsTable.reloadData()
     }
 
+    @objc private func episodeTranscriptAvailabilityChanged(notification: NSNotification) {
+        guard let episodeUuid = notification.userInfo?["episodeUuid"] as? String,
+              let isAvailable = notification.userInfo?["isAvailable"] as? Bool,
```

Comment:
Not related to this PR: I'm wondering if we can make these keys global as we re-use them in other places 🤔